File: 20190519python/0455. Assign Cookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        s.sort()
        g.sort()
        rep=0
        i,j=0,0
        while i<len(s) and j<len(g):
            if s[i]>=g[j]:
                rep+=1
                j+=1
            i+=1
        return rep

File: archives/C#/0455. Assign Cookies.cs
public class Solution {
    public int FindContentChildren(int[] g, int[] s) {
        Array.Sort(g);
        Array.Sort(s);
        int rep=0;
        int gstep=0;
        int sstep=0;
        while(gstep<g.Length && sstep<s.Length){
            if(s[sstep]>=g[gstep]){
                rep++;
                sstep++;
                gstep++;
            }
            else{
                sstep++;
            }
        }
        return rep;
    }
}
